src/lib.rs

```diff
@@ -8,18 +8,22 @@ use std::ffi::c_void;
 use std::io::Result;
 use std::sync::mpsc::{self, Receiver};
 
+use mime::{AllowedMimeTypes, AsMimeTypes, MimeType};
```

Comment:
local crate exports go below the `mod *;` this place is reserved for the outsiders.